WordPress/Sniffs/Files/FileNameSniff.php

```diff
@@ -43,8 +43,12 @@ public function process( PHP_CodeSniffer_File $phpcsFile, $stackPtr ) {
 
 		if ( false !== strpos( $fileName, '_' ) ) {
 			$expected = str_replace( '_', '-', $fileName );
-			$error    = 'Filename "' . $fileName . '" with underscores found; use ' . $expected . ' instead';
-			$phpcsFile->addError( $error, $stackPtr, 'UnderscoresNotAllowed' );
+			$error    = 'Filename "%s" with underscores found; use %s instead';
```

Comment:
We could. I generally try to stick to the "minimal change" principle, i.e. "Don't change more than what you are fixing.", but agree that the variable definitions in this case aren't really needed (goes for all three, not just the variable I added).